### ISSUE TITLE
Exception when editing an account 

### ISSUE THREAD

**Issue by Rafiot** [MISP/MISP#611]
On a fresh install, when I try to edit an account: 
```
Request URL: /admin/users/login
Stack Trace:
#0 /var/www/MISP/app/Lib/cakephp/lib/Cake/Routing/Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#1 /var/www/MISP/app/Lib/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#2 /var/www/MISP/app/webroot/index.php(92): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#3 {main}
2015-08-24 13:14:56 Error: [MissingActionException] Action UsersController::admin_login() could not be found.
Exception Attributes: array (
  'controller' => 'UsersController',
  'action' => 'admin_login',
)
```

Not sure what other information I can give you for helping, but I can mess around a bit is needed :)

**Comment by iglocska** [comment]
You were simply logged out - the problem is that MISP's admin routing will
use the admin prefix (if set, such as it is for user edits) on its
automatic redirect to the login page, which doesn't exist.

tl;dr: MISP thinks that admins have their own login page but they don't,
simply go to https://mymisp and it will route you to the correct login page.

I'll patch this as it is indeed annoying!

**Comment by Rafiot** [comment]
I see, thank you ;)

**Comment by iglocska** [comment]
This was fixed in one of the recent hotfixes, so I'll just close it for now (it will always redirect you to the correct non-admin login).
